backend/capellacollab/core/database/migration.py

```diff
@@ -97,3 +108,29 @@ def create_tools(db):
     tools.create_nature(db, papyrus.id, "UML 2.5")
     tools.create_nature(db, papyrus.id, "SysML 1.4")
     tools.create_nature(db, papyrus.id, "SysML 1.1")
+
+
+def create_repositories(db):
+    LOGGER.info("Initialized repositories")
+    tool = tools.get_tool_by_name(db, "Capella")
+    version = tools.get_version_by_name(db, tool, "5.2")
```

Comment:
```suggestion
    version = tools.get_version_by_name(db, tool, "5.2.0")
```

I've changed it in `v2/backups` to `5.2.0` to have semantic versioning.

Reply:
Then it will be changed by the merge of `v2/backups`, why would I need to change it there?